src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/InputDoWhile.java

```diff
@@ -0,0 +1,19 @@
+package com.puppycrawl.tools.checkstyle.checks.whitespace;
+
+public class InputDoWhile {
+
+    boolean condition() {
+        return false;
+    }
+
+    void testDoWhile() {
+        //Valid
```

Comment:
please remove such comment in all Input files. You do violation on certain line , not a structure.